ftest/nightwatchjs/README.md

```diff
@@ -0,0 +1,9 @@
+# Nuxeo Nightwatch Sample
+
```

Comment:
There is no usage instruction, it's a really simple library to use.

I will try to find something without copy/pasting and without duplicating their website content/documentation...